mimesis/enums.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+from random import choice
+
+
+class PortRange(Enum):
+    DEFAULT = (1, 65535)
+    WELL_KNOWN = (1, 1023)
+    EPHEMERAL = (49152, 65535)
+    REGISTERED = (1024, 49151)
+
+
+class Gender(Enum):
+    FEMALE = 'female'
+    MALE = 'male'
+    F = FEMALE
```

Comment:
I'm not sure that is good idea, but short alias can be useful sometime.

Reply:
No, I don't think so. There is autocomplete for this kind of things.
But for me it would not be clear: is `F` strictly the same as `FEMALE`?

Reply:
@sobolevn I'll remove ones.